src/helperFunctions/toast.ts

```diff
@@ -0,0 +1,12 @@
+import { toast } from 'react-toastify';
+
+type ToastType = 'info' | 'success' | 'warning' | 'error' ;
+
```

Comment:
Can we please add [TypeScript enums](https://www.typescriptlang.org/docs/handbook/enums.html#string-enums)?
Also, export it